internal/ui/login/static/templates/register_org.html

```diff
@@ -0,0 +1,72 @@
+{{template "main-top" .}}
+
+<h1>{{t "RegistrationOrg.Title"}}</h1>
+<p>{{t "RegistrationOrg.Description"}}</p>
+
+<form action="{{ orgRegistrationUrl }}" method="POST">
+
+    {{ .CSRF }}
+
+    <input type="hidden" name="authRequestID" value="{{ .AuthReqID }}" />
+
+    <div class="fields">
+        <div class="field">
+            <label class="label" for="orgname">{{t "RegistrationOrg.OrgName"}}</label>
+            <input class="input" type="text" id="orgname" name="orgname" value="{{ .OrgName }}" autofocus required>
+        </div>
+
+        {{if .UserLoginMustBeDomain}}
+            <div class="field">
+                <label class="label" for="username">{{t "RegistrationOrg.Username"}}</label>
+                <span id="loginname">
+                    <input data-iam-domain="{{ .IamDomain }}"
+                       class="input" type="text" id="username" name="username"
+                       autocomplete="user-name" value="{{ .Username }}" autofocus required>
```

Comment:
```suggestion
                       autocomplete="username" value="{{ .Username }}" autofocus required>
```

Reply:
not sure if we should remove autocomplete: the input is only for the username and not loginname